sql-infra: add resource control feature (#12353)
pingcap/docs: grafana-resource-control-dashboard.md

> ---

Author: Added

title: Key Monitoring Metrics of Resource Control
summary: Learn some key metrics displayed on the Grafana Resource Control dashboard.
---

# Key Monitoring Metrics of Resource Control

If you use TiUP to deploy the TiDB cluster, the monitoring system (Prometheus & Grafana) is deployed at the same time. For more information, see [Overview of the Monitoring Framework](/tidb-monitoring-framework.md).

> The Grafana dashboard is divided into a series of sub dashboards which include Overview, PD, TiDB, TiKV, Node\_exporter, Disk Performance, and Performance\_overview. A lot of metrics are there to help you diagnose.

Reviewer: ```suggestion
The Grafana dashboard is divided into a series of sub dashboards which include Overview, PD, TiDB, TiKV, Node\_exporter, Disk Performance, and Performance\_overview.
```

> You can get an overview of the resource control status from the PD dashboard, where the key metrics are displayed. This document provides a detailed description of these key metrics.

Reviewer: ```suggestion
If your cluster has used the [Resource Control](tidb-resource-control.md) feature, you can get an overview of the resource consumption status from the Resource Control dashboard.
```

> This document describes some key monitoring metrics displayed on the Resource Control Dashboard.

Reviewer: ```suggestion
This document describes some key monitoring metrics displayed on the Resource Control dashboard.
```

> ## Resource Unit

Reviewer 2: ```suggestion
## Request Unit
```

> - RU: the [Request Unit (RU)](/tidb-resource-control.md#what-is-request-unit-ru) consumption information of each resource group is calculated in real time. `total` is the sum of the Request Units consumed by all Resource Groups. The Request Unit consumption of each resource group should be equal to the sum of its read consumption (Read Request Unit) and write consumption (Write Request Unit).

Reviewer: ```suggestion
- RU: the [Request Unit (RU)](/tidb-resource-control.md#what-is-request-unit-ru) consumption information of each resource group, calculated in real time. `total` is the sum of the Request Units consumed by all Resource Groups. The Request Unit consumption of each resource group should be equal to the sum of its read consumption (Read Request Unit) and write consumption (Write Request Unit).
```

> - RU Per Query: the average number of Request Units consumed by each SQL statement per second. It is obtained by dividing the above Request Unit by the number of SQL statements executed per second.

Reviewer: ```suggestion
- RU Per Query: the average number of Request Units consumed by each SQL statement per second. It is obtained by dividing the above RU metric by the number of SQL statements executed per second.
```

> - RRU: the Read Request Unit consumption information of each resource group is calculated in real time. `total` is the sum of the Read Request Units consumed by all Resource Groups.

Reviewer: ```suggestion
- RRU: the Read Request Unit consumption information of each resource group, calculated in real time. `total` is the sum of the Read Request Units consumed by all Resource Groups.
```

> - RRU Per Query: the average number of Read Request Units consumed by each SQL statement per second. It is obtained by dividing the above Read Request Unit by the number of SQL statements executed per second.

Reviewer: ```suggestion
- RRU Per Query: the average number of Read Request Units consumed by each SQL statement per second. It is obtained by dividing the above RRU metric by the number of SQL statements executed per second.
```

> - WRU: the Write Request Unit consumption information of each resource group,  calculated in real time. `total` is the sum of the Write Request Units consumed by all Resource Groups.

Reviewer: ```suggestion
- WRU: the Write Request Unit consumption information of each resource group, calculated in real time. `total` is the sum of the Write Request Units consumed by all Resource Groups.
```

> - WRU: the Write Request Unit consumption information of each resource group is calculated in real time. `total` is the sum of the Write Request Units consumed by all Resource Groups.

Reviewer: ```suggestion
- WRU: the Write Request Unit consumption information of each resource group,  calculated in real time. `total` is the sum of the Write Request Units consumed by all Resource Groups.
```

> - WRU Per Query: the average number of Write Request Units consumed by each SQL statement per second. It is obtained by dividing the above Write Request Unit by the number of SQL statements executed per second.

Reviewer: ```suggestion
- WRU Per Query: the average number of Write Request Units consumed by each SQL statement per second. It is obtained by dividing the above WRU metric by the number of SQL statements executed per second.
```
